nav_msgs/OccupancyGrid occupancy_map
---
bool success
tuw_multi_robot_msgs/Graph voronoi_map